hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -43,12 +49,11 @@
  * it is null to save space.
  */
 @SuppressWarnings("checkstyle:methodcount")
-public class RingbufferContainer implements IdentifiedDataSerializable {
+public class RingbufferContainer<T> implements IdentifiedDataSerializable, Notifier, Versioned {
```

Comment:
As far as I know, `Notifier ` works only for operations. What does it mean here?